dev/AppNotifications/AppNotifications.idl

```diff
@@ -109,6 +109,10 @@ namespace Microsoft.Windows.AppNotifications
     [contract(AppNotificationsContract, 1)]
     runtimeclass AppNotificationManager
     {
+        // Checks to see if the APIs are supported for the Desktop app
+        // Certain scenarios are unsupported by AppNotifications
+        static Boolean IsSupported();
```

Comment:
New APIs need to be in a new contract in order to not breaking the previous vtable.